packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -223,7 +226,12 @@ export class MediaPlaceholder extends Component {
 	}
 
 	renderDropZone() {
-		const { onHTMLDrop = noop } = this.props;
+		const { disableDropZone, onHTMLDrop = noop } = this.props;
+
+		if ( disableDropZone ) {
+			return <Fragment />;
```

Comment:
I think you can just return `null` here instead of having to use a `Fragment`

Reply:
ee1b23822b215b587289570dbd57112f11c14355